### PR TITLE
Connorwon 222 alumni vs member separation

### PR DESCRIPTION
**Pull request by ConnorWon**
## Description

Backend portion of the alumni vs non-alumni member separation

### Changes

* Added new member endpoint which gets members apart of a specific project
* Got rid of FrontEndAPI checking in get request member controllers
* Updated member model to include an isAlumni field and got rid of redundant 'Message' field
* Added and updated member controller tests based on changes

### Updates

[Please enter a brief summary of updates to this PR during code review - if any]

## Checklist

- [x] Ran `npm run lint:fix` to format code
- [x] Assigned PR to all authors
- [x] Requested at least one reviewer

### CODE REVIEW

**Comment by ConnorWon** [comment]
[project members vs alumni separation](https://trello.com/c/KxCNxlPq/222-project-members-vs-alumni-separation)

**Comment by ConnorWon** [comment]
[project members vs alumni separation](https://trello.com/c/KxCNxlPq/222-project-members-vs-alumni-separation)

File: test/unit/controllers/memberControllerTest.js
Comment (ElisaHu1) [inline test/unit/controllers/memberControllerTest.js:1]: I LOVE you adding a unit test for this!

**Review by ryanyae — CHANGES_REQUESTED** [review]
Just as a safe measure we should just add some dummy data to show how it would look like in prod and before we merge this in we should just delete the dummy data

**Comment by ConnorWon** [comment]
> Just as a safe measure we should just add some dummy data to show how it would look like in prod and before we merge this in we should just delete the dummy data

Attached two videos, each showing what the project and people page would look like with dummy data included

https://github.com/UBC-VCL/VCL-content-platform-server/assets/120760211/a77edf24-263b-4b05-8b74-e66ae6e93e47


https://github.com/UBC-VCL/VCL-content-platform-server/assets/120760211/5a631b8a-ab89-41b3-9819-4b23211ed4bf

**Review by ElisaHu1 — APPROVED** [review]

**Review by ryanyae — CHANGES_REQUESTED** [review]
Looks good, @ElisaHu1 I really like Connor's implementation of the team members better than that of Jasmine's. I think we should merge this ticket in and close Jasmine's ticket regarding the team members mobile view ticket. @ConnorWon I would also like to add that we dont need the breadcrumbs at the top anymore, do you also think you could add a max-width property to the alumni button (image for reference below).

image

**Comment by ConnorWon** [comment]
> Looks good, @ElisaHu1 I really like Connor's implementation of the team members better than that of Jasmine's. I think we should merge this ticket in and close Jasmine's ticket regarding the team members mobile view ticket. @ConnorWon I would also like to add that we dont need the breadcrumbs at the top anymore, do you also think you could add a max-width property to the alumni button (image for reference below).
> 
> EDIT: I realize this is the backend PR, but pretend like this request is for the frontend PR.
> 
> image

Removed the breadcrumbs and added the max-width

**Review by ryanyae — APPROVED** [review]

**Comment by ElisaHu1** [comment]
> Looks good, @ElisaHu1 I really like Connor's implementation of the team members better than that of Jasmine's. I think we should merge this ticket in and close Jasmine's ticket regarding the team members mobile view ticket. @ConnorWon I would also like to add that we dont need the breadcrumbs at the top anymore, do you also think you could add a max-width property to the alumni button (image for reference below).
> 
> EDIT: I realize this is the backend PR, but pretend like this request is for the frontend PR.
> 
> image

yes ryan you can go ahead and close jasmine ticket

**Pull request merged**